pyro/distributions/torch.py

```diff
@@ -101,6 +101,19 @@ def log_prob(self, value):
         return (-value - 1) * torch.nn.functional.softplus(self.logits) + self.logits
 
 
+class LogNormal(torch.distributions.LogNormal, TorchDistributionMixin):
+    def __init__(self, loc, scale, validate_args=None):
+        base_dist = Normal(loc, scale)
+        # This differs from torch.distributions.LogNormal only in that base_dist is
+        # a pyro.distributions.Normal rather than a torch.distributions.Normal.
+        super(torch.distributions.LogNormal, self).__init__(
+            base_dist, torch.distributions.transforms.ExpTransform(), validate_args=validate_args)
+
+    def expand(self, batch_shape, _instance=None):
+        new = self._get_checked_instance(LogNormal, _instance)
+        return super(torch.distributions.LogNormal, self).expand(batch_shape, _instance=new)
```

Comment:
Can you confirm this results in a `.base_dist` of the correct `Normal` type?

Reply:
Yes, it is
```python
import torch
from pyro.distributions.torch import Normal, TorchDistributionMixin

class LogNormal(torch.distributions.LogNormal, TorchDistributionMixin):
    def __init__(self, loc, scale, validate_args=None):
        base_dist = Normal(loc, scale)
        super(torch.distributions.LogNormal, self).__init__(
            base_dist, torch.distributions.transforms.ExpTransform(), validate_args=validate_args)

    def expand(self, batch_shape, _instance=None):
        new = self._get_checked_instance(LogNormal, _instance)
        return super(torch.distributions.LogNormal, self).expand(batch_shape, _instance=new)

x = LogNormal(0, 1).expand([3])
assert repr(type(x.base_dist)) == "<class 'pyro.distributions.torch.Normal'>"
```